ghcide/src/Development/IDE/Core/Service.hs

```diff
@@ -46,10 +48,11 @@ initialise :: Config
            -> HieDb
            -> IndexQueue
            -> IO IdeState
-initialise defaultConfig mainRule lspEnv logger debouncer options vfs hiedb hiedbChan =
+initialise defaultConfig mainRule dynFlagsMods lspEnv logger debouncer options vfs hiedb hiedbChan =
```

Comment:
Instead of an additional argument here, can we have this as a field in the `Options`?

Reply:
Options already has a `optCustomDynFlags` field (added by you in b57a942e2). As best I can tell, today it's only ever `id`, but this seems like a good field to replace with the `DynFlagsModifications`.

Reply:
Sounds good